WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -70,6 +73,46 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
   result['StdErr'] = error
   return result
 
+def getARCPilotOutput( proxy, pilotRef ):
+  tmp_dir = mkdtemp()
+  myce = pilotRef.split(":")[1].strip("/")
+  gridEnv = getGridEnv()
+  mySite = getCESiteMapping()['Value'][myce]
+  WorkDB = gConfig.getValue('Resources/Sites/LCG/' + mySite + '/CEs/' + myce + '/JobListFile')
```

Comment:
Hi Chris,

Do you mean something like

   WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))

As a non-expert, why is this superior?

Thanks,
Raja.

On 01/04/15 16:55, chaen wrote:
> In WorkloadManagementSystem/Service/WMSUtilities.py <https://github.com/DIRACGrid/DIRAC/pull/2324#discussion_r27582878>:
>
>> @@ -70,6 +73,46 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
>>    result['StdErr'] = error
>>    return result
>>
>> +def getARCPilotOutput( proxy, pilotRef ):
>> +  tmp_dir = mkdtemp()
>> +  myce = pilotRef.split(":")[1].strip("/")
>> +  gridEnv = getGridEnv()
>> +  mySite = getCESiteMapping()['Value'][myce]
>> +  WorkDB = gConfig.getValue('Resources/Sites/LCG/' + mySite + '/CEs/' + myce + '/JobListFile')
>
> It's maybe being picky, but to avoid nasty surprises, it's better to use os.path.join for that kind of path operation
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/DIRACGrid/DIRAC/pull/2324/files#r27582878>.
>